deploy/custom/configuration.yaml

```diff
@@ -143,52 +143,38 @@
         {
           "metric_filters": [
```

Comment:
have Charlie look at these, if he hasn't already, all these rules came from his dashboard requirements.

Reply:
Those are the filters that he created